docs/pages/enterprise/sso/one-login.mdx

```diff
@@ -19,7 +19,7 @@ like:
   type="warning"
   title="Version Warning"
 >
-  This guide requires an Enterprise edition of Teleport. The Open Source Edition of Teleport only supports [Github](../../admin-guide.mdx#github-oauth-20) as an SSO provider.
+  This guide requires an Enterprise edition of Teleport.
 </Admonition>
```

Comment:
Nit: This "requires enterprise version" admonition may be a good candidate to move into a partial since it's used in at least 6 or 7 places (that I counted). This would also address a little bit inconsistent language - right now it says "commercial edition" version in some and "Enterprise edition" in others. Up to you though.

Reply:
yes, I'm working on it with docs engineers